config/etcd-io/org.yaml

```diff
@@ -87,8 +87,6 @@ teams:
     members:
     - ahrtr
     - jmhbnz
-    - mitake
-    - ptabor
```

Comment:
We need to remove the retired maintainers from Kubernetes/orgs/sig-etcd/*.yaml as well.

Reply:
OOPS, it's exactly what this PR is doing. I was thinking this PR is changing etcd-io/website 's OWNER file:)

Please ignore this comment